app/views/smart_answers/index.html.erb

```diff
@@ -1,8 +1,13 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <%= render "govuk_publishing_components/components/title", {
-      title: "Smart Answers",
-    } %>
+    <div class="govuk-!-margin-top-8">
```

Comment:
Could the `margin-top` go on the column itself, rather than adding an extra element?